internal/worker/json.go

```diff
@@ -18,6 +18,7 @@ type OSBuildJob struct {
 	Targets         []*target.Target `json:"targets,omitempty"`
 	ImageName       string           `json:"image_name,omitempty"`
 	StreamOptimized bool             `json:"stream_optimized,omitempty"`
+	Exports         []string         `json:"export_stages,omitempty"`
```

Comment:
We have to be careful when extending this API, keeping in mind that composer and workers will not be updated in lockstep. Adding a field is not a problem, as long as we can handle the case when composer passes it to a worker that does not expect it, and that workers can deal with not being passed it.